src/components/ArticlesList.jsx

```diff
@@ -4,19 +4,19 @@ import ArticleCard from './ArticleCard.jsx'
 
 function ArticlesList() { 
     const [articles, setArticles] = useState([])
+    const [avotes, setAVotes]=useState(0)
     
```

Comment:
I'd like to see a loading state here t provide visual feedback to the user on slow load